minio/Dockerfile

```diff
@@ -49,32 +49,21 @@ RUN apk update && apk --no-cache upgrade && \
 
 WORKDIR /app
 
-RUN \
-    # Download minio along host architecture
-    ARCH=$(uname -m); \
-    if [ "$ARCH" = "x86_64" ]; then \
-        ARCH=amd64; \
-    elif [ "$ARCH" = "aarch64" ]; then \
-        ARCH=arm64; \
-    else \
-        echo "Unsupported architecture: $ARCH"; \
-        exit 1; \
-    fi; \
-    # Fetch Minio
-    wget $(echo "https://dl.min.io/server/minio/release/linux-$ARCH/minio") && \
-    # Apply proper rights on executable
-    chmod +x minio && \ 
-    # Create folder for volume data
-    mkdir /data
-
 # Define defaults minio root credentials
 ENV MINIO_ROOT_USER=minioadmin
 ENV MINIO_ROOT_PASSWORD=minioadmin
+ENV MINIO_BROWSER_LOGIN_ANIMATION=false
```

Comment:
Try to install `mc` cli tool inside container
- https://min.io/docs/minio/linux/reference/minio-mc.html#install-mc
- https://pkgs.alpinelinux.org/packages?name=minio-client&branch=v3.18&repo=&arch=&maintainer=
  - create alias mcli => mc